common/lib/capa/capa/responsetypes.py

```diff
@@ -248,7 +248,9 @@ def render_html(self, renderer, response_msg=''):
           - response_msg: a message displayed at the end of the Response
         """
         # render ourself as a <span> + our content
-        tree = etree.Element('span')
+        tree = etree.Element('section')
+        tree.set('class', 'wrapper-problem-response')
+        tree.set('tabindex', '-1')
```

Comment:
If you have a heading as the first child with an `id`, you can use `aria-labelledby="said id"`. If there won't be a heading, you should use `aria-label`.